lib/ansible/modules/network/vyos/vyos_interface.py

```diff
@@ -244,7 +248,7 @@ def map_config_to_obj(module):
                 if match:
                     param = match.group(1)
                     if param == 'description':
-                        match = re.search(r'description (\S+)', line, re.M)
+                        match = re.search(r'description \s*([^\n\r]*)', line, re.M)
```

Comment:
@rohitthakur2590 just curious why do we need to modify the regex?

Reply:
@NilashishC  earlier logic was picking only one word after description once it will find space  .Lets say we have 
line =  "set interfaces ethernet eth1 description 'ETH1 is for downlink  be set'"  then per previous regex only   ETH1  will be considered .

Reply:
Oh okay, maybe description (.+) also does the trick?